lib/container/ContainerEntryModule.js

```diff
@@ -230,7 +230,7 @@ class ContainerEntryModule extends Module {
 				`if (!${RuntimeGlobals.shareScopeMap}) return;`,
 				`var name = ${JSON.stringify(this._shareScope)}`,
 				`var oldScope = ${RuntimeGlobals.shareScopeMap}[name];`,
-				`if(oldScope && oldScope !== shareScope) throw new Error("Container initialization failed as it has already been initialized with a different share scope");`,
+				'if(oldScope && oldScope !== shareScope) throw new Error("Container initialization failed as it has already been initialized with a different share scope");',
```

Comment:
Bugfix in [@stylistic/quotes rule](https://github.com/webpack/webpack/actions/runs/11320033480/job/31476957279?pr=18856) - https://github.com/eslint-stylistic/eslint-stylistic/pull/544